assets/constants.ts

```diff
@@ -414,6 +414,9 @@ export default {
   "ALLOWED_STORYNODE_STATUS": ["Draft", "Ready To Publish", "Published"],
```

Comment:
This should have been caught earlier, but "STORY NODE" is two words, here and in the next few lines.

Reply:
Done